style.css

```diff
@@ -1,336 +1,704 @@
-}
+/* ************************ desktop and laptop screen sizes **************** */
 
-.headline-summary {
-  width: 327px;
-  height: 164px;
-  text-align: center;
-  font-weight: 400;
-  font-size: 20px;
-  padding-top: 85px;
-  line-height: 28px;
-  color: #172b4d;
-}
+@media (min-width: 768px) {
+  .headline {
+    background-image: url(images/Header-llustration-desktop.svg);
+    height: 1400px;
+    background-size: contain;
+    background-repeat: no-repeat;
+  }
 
-.social-links {
-  display: flex;
-  justify-content: space-evenly;
-  padding-top: 60px;
-  width: 180px;
-  padding-left: 100px;
-  font-size: 20px;
-}
+  .mobile-nav {
+    display: none;
+  }
 
-.headline-button {
-  padding-top: 35px;
-}
+  .desktop-nav {
+    display: flex;
+    justify-content: center;
+    font-size: 22px;
+    color: #172b4d;
+    padding-top: 2em;
+  }
 
-.list {
-  list-style: none;
-}
+  .nav-item {
+    padding-left: 1em;
+    text-decoration: none;
+  }
 
-a {
-  color: #172b4d;
-}
+  .headline-title {
+    font-weight: 700;
+    font-size: 40px;
+    text-align: center;
+    padding-top: 40px;
+    padding-right: 100px;
+    color: #172b4d;
+  }
 
-.grid-container {
-  display: grid;
-  gap: 16px;
-  margin: 20px 25px;
-}
+  .t2 {
+    color: #36b37e;
+  }
 
-.grid-items {
-  width: 320px;
-  height: 474px;
-  background-color: #ebf0ee;
-  border: #cfd0d1 solid 1px;
-  border-radius: 8px 8px 8px 8px;
-}
+  .headline-summary {
+    text-align: center;
+    font-size: 20px;
+    line-height: 35px;
+    color: #172b4d;
+    margin-left: 20px;
+    margin-right: 50px;
+  }
 
-.description {
-  height: 225px;
-  margin-top: 249px;
-  border-radius: 8px 8px 8px 8px;
-  background: #fff;
-}
+  .social-links {
+    display: flex;
+    justify-content: center;
+    padding-top: 30px;
+    width: 280px;
+    padding-left: 22%;
+    font-size: 25px;
+    gap: 30px;
+  }
 
-.new {
-  color: #3a4a42;
-  text-align: center;
-  padding-top: 15px;
-  font-size: 26px;
-  line-height: 32px;
-}
+  .headline-button {
+    padding-top: 35px;
+    padding-top: 50px;
+    margin-left: 100px;
+  }
 
-.btn-btn {
-  height: 35px;
-  font-size: 15px;
-  border-radius: 4px 4px 4px 4px;
-  margin: 3px;
-  margin-top: 10px;
-  color: #3a4a42;
-  background: #ebf0ee;
-  border: none;
-}
+  /* ************************ section One Above **************** */
 
-.btn {
-  margin-top: 15px;
-  height: 50px;
-  width: 128px;
-  background: #36b37f;
-  border-radius: 6px 6px 6px 6px;
-  border: none;
-  display: flex;
-  color: #fff;
-  font-size: 19px;
-  margin-left: 100px;
-  padding-top: 14px;
-  padding-left: 13px;
-}
+  .section-tag {
+    font-weight: 700;
+    font-size: 25px;
+    text-align: center;
+    margin-bottom: 100px;
+  }
 
-.btn-btn:hover {
-  background: #36b37e;
-  color: #fff;
-}
+  .section-title {
+    color: #172b4d;
+  }
 
-.btn-btn:active {
-  background: #008552;
-}
+  .high-container {
+    display: flex;
+    justify-content: center;
+  }
 
```

Comment:
Kindly adjust the works section to match your chosen design template. A look at the design shows that there are three cards per row.

Also, kindly maintain the padding of the container of the works and the gap between individual works.

Your design:
![image](https://user-images.githubusercontent.com/20554583/200074317-fda9ce5a-f648-498e-a6a7-3f220f518cad.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/200074380-b6fb7314-fe9a-441a-9940-49eb413cca25.png)